spring-core/src/test/java/org/springframework/core/SimpleAliasRegistryTests.java

```diff
@@ -60,4 +60,26 @@ void aliasChainingWithMultipleAliases() {
 		assertThat(registry.hasAlias("real_name", "alias_c")).isTrue();
 	}
 
+	@Test
+	void testOtherMethod() {
```

Comment:
Please introduce three dedicated `@Test` methods for the `SimpleAliasRegistry` methods you wish to test -- for example, `hasAlias()`, `isAlias()`, and `getAliases()` instead of a single `testOtherMethod()` method.

Reply:
I see. Thank you for your feedback!